scripts/SREP/delete-olm-operator/metadata.yaml

```diff
@@ -0,0 +1,32 @@
+file: script.sh
+name: delete-olm-operator 
+description: Delete all of the olm data in a given namespace (if syncsets are working, it will recreate)
+author: gerogettica
+allowedGroups: 
+  - SREP
+  - CSSRE
+  - CEE
```

Comment:
Do we wanna CEE do this operation?

Reply:
We can only keep SREP here, as this used by SREP only atm.

Reply:
Done